src/Services/LoggingService.php

```diff
@@ -104,6 +116,8 @@ private function getPlentyMarketLoggerInstance(string $method, string $reference
   }
 
   /**
+   * Maps data from the PlentyMarket designed associative array to values of an array to be used by our different log methods
```

Comment:
I think your description is backwards.
Doesn't this map the data from the associative array that Wayfair provides to the PlentyMarkets logging API's inputs?